render/render.go

```diff
@@ -211,8 +214,8 @@ func (p TextRenderer) Render(w io.Writer, x ast.Expr) (mErr error) {
 	switch t := x.(type) {
 	case *ast.ParsedText:
 		for _, v := range t.Values {
-			if err := p.Render(w, v); err != nil {
-				return err
+			if mErr = p.Render(w, v); mErr != nil {
```

Comment:
ok, great, thanks for the pointer. I interpreted this as: use `mErr` when you don't return immediately, and `err` otherwise.

I've adjusted the code accordingly, let me know if you'd like me to remove all `mErr` uses instead.